ScreeningEncounterForm.html

```diff
@@ -0,0 +1,1209 @@
+	{"label": "Kapretwa", "val": "173"},
+	{"label": "Sukwo", "val": "174"},
+	{"label": "Muroki", "val": "172"},
+	{"label": "Koikoi", "val": "175"},
+	{"label": "Bidii", "val": "331"},
+	{"label": "Chepng'oror", "val": "233"},
+	{"label": "Kipkabus", "val": "227"},
+	{"label": "Olare", "val": "219"},
+	{"label": "Pioneer", "val": "335"},
+	{"label": "Kibulgeng", "val": "337"},
+	{"label": "Soy", "val": "338"},
+	{"label": "Kapsoya", "val": "144"},
+	{"label": "Mlimani", "val": "340"},
+	{"label": "Langas", "val": "342"},
+	{"label": "Munyaka", "val": "344"},
+	{"label": "Kipsigak", "val": "156"},
+	{"label": "Lemorong'eny", "val": "235"},
+	{"label": "Ziwa", "val": "28"},
+	{"label": "Kapsang'", "val": "236"},
+	{"label": "Kong'asis", "val": "237"},
+	{"label": "Segero", "val": "238"},
+	{"label": "Sinoko", "val": "240"},
+	{"label": "Webuye District Hosp", "val": "8"},
+	{"label": "East Karateng", "val": "151"},
+	{"label": "Newa", "val": "152"},
+	{"label": "Marera", "val": "150"},
+	{"label": "South Kapuonja", "val": "153"},
+	{"label": "North Kapuonja", "val": "154"},
+	{"label": "Sunga", "val": "339"},
+	{"label": "West Karateng", "val": "155"},
+    {"label": "Kokwet", "val": "114"},
+    {"label": "Kosirai", "val": "159"},
+    {"label": "Mutwot", "val": "160"},
+    {"label": "Mosoriot", "val": "2"},
+    {"label": "Birbiriet", "val": "111"},
+    {"label": "Itigo", "val": "112"},
+    {"label": "Chepterit", "val": "133"},
+    {"label": "Ngechek", "val": "115"},
+    {"label": "Lelmokwo", "val": "113"},
+    {"label": "sigot", "val": "119"},
+    {"label": "Chepkemel ", "val": "122"},
+    {"label": "Cheplaskei", "val": "118"},
+    {"label": "Tuigoin", "val": "157"},
+    {"label": "Sugoi A", "val": "120"},
+    {"label": "Sugoi B", "val": "121"},
+    {"label": "Leseru", "val": "158"},
+    {"label": "MurgoHillls", "val": "136"},
+    {"label": "Murgusi", "val": "117"},
+    {"label": "Sambut", "val": "127"},
+    {"label": "Sosiani", "val": "129"},
+    {"label": "Turbo/Kaptebee", "val": "155"},
+    {"label": "Chepsaita", "val": "60"},
+    {"label": "Cheramei", "val": "116"},
+    {"label": "Ngenyilel", "val": "128"},
+    {"label": "Osorongai", "val": "137"},
+    {"label": "Sokyot", "val": "154"}
+    ];
+    $('#encounter\\.location_id').autocomplete({
+        source: locationName,
+        create: function (event, ui) {
+            var location_val = $('input[name="encounter\\.location_id"]').val();
+            $.each(locationName, function (i, elem) {
+                if (elem.val == location_val) {
+                    $('#encounter\\.location_id').val(elem.label)
+                }
+            });
+        },
+        select: function (event, ui) {
+            $('input[name="encounter\\.location_id"]').val(ui.item.val);
+            $('#encounter\\.location_id').val(ui.item.label);
+            return false;
+        }
+    });
+
+    var provider = [
+    {"label": "Christopher J Mwenengo", "val": "2281-4"},
```

Comment:
I think we should look into the option of having separate javascript file just for ampath / lark providers and locations.